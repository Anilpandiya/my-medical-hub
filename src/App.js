import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
import Dashboard from "./Components/Dashboard/Dashboard";
import Doctor from "./Components/Doctor/Doctor";
import Patient from "./Components/Patient/Patient";

function App() {
  return (
    <Router>
      <Routes>
        <Route exact path="/" element={<Dashboard/>} />
        <Route path="/doctor" element={<Doctor/>} />
        <Route path="/patient" element={<Patient/>} />
      </Routes>
    </Router>
  );
}

export default App;
