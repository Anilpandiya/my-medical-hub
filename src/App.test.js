import { render, screen } from '@testing-library/react';
import App from './App';

test('renders My Medical HUB text', () => {
  render(<App />);
  expect(screen.getByText(/My Medical HUB/i));
});

test('renders buttons for Doctor & Patient', () => {
  render(<App />);
  expect(screen.getAllByRole('button'));
});
