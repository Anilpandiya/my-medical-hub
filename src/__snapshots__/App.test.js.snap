// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="dashboard"
>
  My Medical HUB
  <div>
    <button
      className=" mbsc-reset mbsc-font mbsc-button mbsc-ios mbsc-ltr mbsc-button-outline mbsc-button-primary"
      onClick={[Function]}
    >
      Doctor
    </button>
    <button
      className=" mbsc-reset mbsc-font mbsc-button mbsc-ios mbsc-ltr mbsc-button-outline mbsc-button-primary"
      onClick={[Function]}
    >
      Patient
    </button>
  </div>
</div>
`;
